server/data/data-jmap/src/main/java/org/apache/james/jmap/api/preview/Preview.java

```diff
@@ -59,6 +60,7 @@ private static String truncateToMaxLength(String body) {
         this.value = value;
     }
 
+    @JsonValue
```

Comment:
Domain objects must keep agnostic to frameworks (so it means no jackson annotation).
There are various way to fix this:
* calling asString to include a string version in the DTO
* having a PreviewDTO object with the annotation
* using explicit json definition with Jackson JsonObject or Scala Writers